features/step_definitions/work_steps.rb

```diff
@@ -160,6 +160,13 @@
   end
 end
 
+Given /^the work "([^\"]*)" in the unrevealed collection "([^\"]*)"$/ do |work, collection|
```

Comment:
FYI, quote marks aren't special in Regex and therefore don't need to be escaped, so the placeholders could be `([^"]*)` (meaning any number of characters that are not a quote) or for that matter just `(.*)` (meaning any number of characters until you meet whatever is after this, which in this case will be the first quote after the placeholder)